.github/workflows/test-PRs.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test PRs
+
+on:
+  push:
+    branches: # Todo: All branches?
+      - 'master'
+      - 'map3d'
+      - 'panda3d_voxel_map'
```

Comment:
Some of these branches don't exist anymore. I think it's best if we have them enabled on all branches, since branch naming is very volatile.